website/content/en/docs/getting-started/users.md

```diff
@@ -28,40 +28,34 @@ Before you start, you must have access to a Kubernetes cluster. For example, you
 
     ```sh
     clusterlink create fabric --name <fabric_name>
     ```
 
-    This command will create the CA files `cert.pem` and `key.pem` in a folder named <fabric_name>. The `--name` option is optional, and by default, "default_fabric" will be used.
-
-1. {{< anchor create-peer-certs >}}Create peer certificates.
+    The ClusterLink fabric is defined as all K8s clusters (peers) that install ClusterLink gateways and can share services between the clusters, enabling communication among those services.
+    This command will create the CA files `cert.pem` and `key.pem` in a directory named <fabric_name>. The `--name` option is optional, and by default, "default_fabric" will be used.
 
-    Create a peer (cluster) certificate:
+1. {{< anchor create-peer-certs >}}Create a peer (cluster) certificate:
 
     ```sh
     clusterlink create peer-cert --name <peer_name> --fabric <fabric_name>
     ```
 
-    This command will create the certificate files `cert.pem` and `key.pem` in a folder named <fabric_name/peer_name>. The `--path <path>` flag can be used to change the folder location.The `--name` option is optional, and by default, "default_fabric" will be used.
+    This command will create the certificate files `cert.pem` and `key.pem` in a directory named <fabric_name>/<peer_name>. The `--path <path>` flag can be used to change the directory location. The `--name` option is optional, and by default, "default_fabric" will be used.
 
-1. {{< anchor install-cl-operator >}}Install ClusterLink.
-
-    To install ClusterLink on the peer, first install the ClusterLink deployment operator.
+1. {{< anchor install-cl-operator >}}Install ClusterLink deployment operator:
 
     ```sh
     clusterlink peer deploy --autostart --name <peer_name> --fabric <fabric_name>
     ```
 
-    This command will deploy the ClusterLink operator on the `clusterlink-operator` namespace and convert the peer certificates to secrets in the namespace.
-    The command assumes that `kubectl` is set to the correct peer (Kubernetes cluster) and that the certificates were created in the local folder. If they were not, use the flag `--path <path>`. The `--fabric` option is optional, and by default, "default_fabric" will be used.
-    The `--autostart` option will deploy the ClusterLink components in the `clusterlink-system` namespace, and the ClusterLink project will start to run in the cluster.
+    This command will deploy the ClusterLink operator on the `clusterlink-operator` namespace and convert the peer certificates to secrets in this namespace.
+    The command assumes that `kubectl` is set to the correct peer (K8s cluster) and that the certificates were created by running the previous command on the same working directory. If they were not, use the flag `--path <path>` for pointing to the working directory that was used in the previous command. The `--fabric` option is optional, and by default, "default_fabric" will be used.
```

Comment:
nits: 
- please break line.
- consistently capitalize K8s (or k8s)
- please confirm the default fabric name
```suggestion
    The command assumes that `kubectl` is set to the correct peer (K8s cluster) and that the certificates were created by running
    the previous command on the same working directory. If they were not, use the `--path <path>` flag to point to the
    working directory that was used in the previous command. The `--fabric` option is optional, and by default, "default_fabric" will be used.
```